### DISCUSSION TITLE
Which A/B to disable new comments layout?

### DISCUSSION THREAD

**Discussion by yodaluca23** [PoomSmart/YTABConfig#29] [Q&A]
Which A/B to disable new comments layout?

![image](https://user-images.githubusercontent.com/67206487/199303105-0acb3169-8f7e-4921-a034-6c7ae22cb63b.jpeg)

**Comment by arichornlover** [comment]
Here you go `DemouiSystemsClientGlobalConfigEnableModernButtonsForNative` toggle this off in “A/B”
`uiSystemsClientGlobalConfigEnableRoundedDialogForNative` toggle this off in “A/B”

**Reply by yodaluca23** [reply]
I turned off the second one, but couldn't find the first one… And it still shows up like I specified in the picture

**Reply by arichornlover** [reply]
I just checked and the first flag I said got removed in the recent YouTube versions, sorry about that.

**Reply by arichornlover** [reply]
I’ll give you a different way to do it, these flags should hopefully to disable the new comments layout. hope these help!
turn off `enableCinematicContainer`
turn off `enableCinematicContainerOnClient`
turn off `uiSystemsClientGlobalConfigEnableModernButtonsForNative`
turn off `uiSystemsClientGlobalConfigEnableModernButtonsForNativeLongTail`
turn off `uiSystemsClientGlobalConfigIosEnableModernTabsForNative`

**Reply by yodaluca23** [reply]
> I’ll give you a different way to do it, these flags should hopefully to disable the new comments layout. hope these help!
> 
> turn off `enableCinematicContainer`
> 
> turn off `enableCinematicContainerOnClient`
> 
> turn off `uiSystemsClientGlobalConfigEnableModernButtonsForNative`
> 
> turn off `uiSystemsClientGlobalConfigEnableModernButtonsForNativeLongTail`
> 
> turn off `uiSystemsClientGlobalConfigIosEnableModernTabsForNative`

Did those, restarted app, and it still shows up how I specified...

**Reply by arichornlover** [reply]
Okay then it’s very weird. I don’t have the new comment section layout and I don’t know what else to do to help you.

**Reply by yodaluca23** [reply]
> Okay then it’s very weird. I don’t have the new comment section layout and I don’t know what else to do to help you.

Thanks for trying anyways…

**Reply by arichornlover** [reply]
There is ways, PoomSmart probably knows more flags to disable the new comment ui more then me
